$NetBSD: patch-aa,v 1.1 1998/08/24 18:11:42 agc Exp $

Work around deficiencies in the Configure script:
+ don't test for stdin being a tty
+ NetBSD's ar ts does generate random libs, but you *STILL* need
to run ranlib, especially on a.out platforms. Comment out ar ts
test - it's just plain wrong.

*** Configure.orig	Sun Feb  9 22:04:23 1992
--- Configure	Sun Mar  8 19:33:26 1998
@@ -28,10 +28,11 @@
 PATH=$PATH:'/bsd4.3/usr/bin:/usr/bsd'
 export PATH || (echo "OOPS, this isn't sh.  Desperation time.  I will feed myself to sh."; sh $0; kill $$)
 
-if test ! -t 0; then
-    echo "Say 'sh Configure', not 'sh <Configure'"
-    exit 1
-fi
+# Neither FreeBSD's ports, or NetBSD's packages, want to be interactive
+#if test ! -t 0; then
+#    echo "Say 'sh Configure', not 'sh <Configure'"
+#    exit 1
+#fi
 
 (alias) >/dev/null 2>&1 && \
     echo "(I see you are using the Korn shell.  Some ksh's blow up on Configure," && \
@@ -872,7 +873,7 @@
 : determine where manual pages are on this system
 echo " "
 case "$sysman" in
-	'') sysman=`loc . /usr/man/man1 /usr/man/man1 /usr/man/mann /usr/man/manl /usr/man/local/man1 /usr/man/u_man/man1 /usr/share/man/man1 /usr/catman/u_man/man1 /usr/man/l_man/man1 /usr/local/man/u_man/man1 /usr/local/man/l_man/man1 /usr/man/man.L`
+	'') sysman=`loc . /usr/local/man/man1 /usr/man/man1 /usr/man/mann /usr/man/manl /usr/man/local/man1 /usr/man/u_man/man1 /usr/share/man/man1 /usr/catman/u_man/man1 /usr/man/l_man/man1 /usr/local/man/u_man/man1 /usr/local/man/l_man/man1 /usr/man/man.L`
     ;;
 esac
 if test -d "$sysman"; then
@@ -1988,12 +1989,12 @@
 : see if ar generates random libraries by itself
 echo " "
 echo "Checking how to generate random libraries on your machine..."
-ar rc ran.a /dev/null
-if ar ts ran.a >/dev/null 2>&1; then
-	echo "ar appears to generate random libraries itself."
-	orderlib=false
-	ranlib=":"
-else
+#ar rc ran.a /dev/null
+#if ar ts ran.a >/dev/null 2>&1; then
+#	echo "ar appears to generate random libraries itself."
+#	orderlib=false
+#	ranlib=":"
+#else
 	if test -f /usr/bin/ranlib; then
 		ranlib=/usr/bin/ranlib
 	elif test -f /bin/ranlib; then
@@ -2009,7 +2010,7 @@
 		orderlib=true
 		ranlib=":"
 	fi
-fi
+#fi
 
 : determine compiler compiler
 case "$bison" in
